examples/headless-example/src/components/App.svelte

```diff
@@ -191,37 +209,37 @@
 
 </script>
 
-{#if $endUserQuery?.data?.id}
-  <div>
-    <div>
-      <h4>{$endUserQuery?.data?.firstName ?? ''} {$endUserQuery?.data?.lastName ?? ''}</h4>
-      <img alt="avatar" src={$endUserQuery?.data?.avatarUrl ?? ''}/>
-    </div>
-      <p>{message}</p>
-  </div>
-{/if}
-
-{#if !$endUserQuery?.data?.id}
-  <SignUp {onSubmit}/>
-{/if}
-{#if $workflow}
-  <Workflow workflow={$workflow}/>
-{/if}
-{#if $endUserQuery?.data?.id && $endUserQuery?.data?.state !== 'PROCESSING' && !$workflow}
-  <button disabled={!$endUserQuery?.data?.id} on:click={$intentQuery.refetch}>Start KYC
-  </button>
-{/if}
-{#if $endUserQuery?.data?.id && $endUserQuery?.data?.state === 'PROCESSING' && !$workflow}
-  <p>
-    We're processing your request.
-  </p>
-{/if}
-
-{#if $endUserQuery?.data?.id && shouldResubmit && !$workflow}
-  <p>
-    You've been requested to re-submit your documents due
-    to {nextWorkflow?.definition?.context?.documentOne?.resubmissionReason?.toLowerCase()?.replace(/_/g, ' ')}
-    . Please click navigate to re-submit your documents.
-  </p>
-  <button on:click={handleResubmit}>Navigate</button>
-{/if}
+<main class="h-full flex flex-col items-center justify-center p-6">
+
+  {#if !endUserId}
+    <SignUp {onSubmit}/>
```

Comment:
looks like we're achieving routing by using if's - do we consider using real routing?

Reply:
Routing wouldn't make sense since it is unrelated to the URL and this is just a demo. I also spoke with Alon on how we need more reliable properties to create conditions with so we won't need as many ifs or at least as complicated.